docs/governance/steering/elections.md

```diff
@@ -0,0 +1,126 @@
+# Common Cloud Controls (CCC) Steering Committee Elections
+
+This document outlines the process for steering committee elections.
+
+## Eligibility
+
+Restrictions on candidacy and voting are in place to ensure that the CCC project is led by individuals who are highly trusted, impactful, and currently active in the community.
+
+### Eligibility for voting
+
+In order to be eligible for voting, an individual must:
+
+- Be an FINOS Member<sup>[1](#footnote1)</sup> who had a substantial contribution to the CCC project within 12 months prior to their nomination. Contributions include GitHub events like creating issues, creating PRs, reviewing PRs, commenting on issues, etc.
```

Comment:
is this any of these, or all of these?

Reply:
Contributions include x, y, z, and someone must have a substantial contribution to be eligible. 

@robmoffat Could you suggest phrasing that helps increase clarity? Currently I'm not seeing the gap.

Reply:
@eddie-knight Community discussed and agree with present wording